### PR TITLE
Mobile View

### PR DESCRIPTION
**Pull request by jamieleegordon**
Login and Sign Up pages

image

image

### CODE REVIEW

**Comment by jamieleegordon** [comment]
image

image

image

File: chatez/src/pages/Chats/Components/ChatFeedProfilePanel/ChatFeedProfilePanel.tsx [hunk 1]
```diff
@@ -49,12 +52,33 @@ const ChatFeedProfilePanel: React.FC<ChatFeedProfilePanelProps> = ({
...
-      }
+      } 
+      ${
+        isCollapsed ? 'ChatFeedProfilePanel-Collapsed' : 'ChatFeedProfilePanel'
+      }`}
     >
+      <div>
+        <IconButton
+          id="CollapseProfilePanel-button"
+          onClick={toggleCollapse} // Toggle collapse on click
```
Comment (Stegi56) [inline chatez/src/pages/Chats/Components/ChatFeedProfilePanel/ChatFeedProfilePanel.tsx:76]: comment not needed here

File: chatez/src/pages/Chats/Components/ChatFeedFriendPanel/ChatFeedFriendPanel.tsx [hunk 2]
```diff
@@ -240,6 +249,13 @@ const ChatFeedFriendPanel: React.FC<ChatFeedFriendPanelProps> = ({
           alt="Display icon"
         />
         {userDisplayName}
+
+        <IconButton
+          id="CollapsePanel-button"
+          onClick={toggleCollapse} // Toggle collapse on click
```
Comment (Stegi56) [inline chatez/src/pages/Chats/Components/ChatFeedFriendPanel/ChatFeedFriendPanel.tsx:255]: comment not needed here

File: chatez/src/pages/Chats/Components/ChatFeedFriendPanel/ChatFeedFriendPanel.css [hunk 3]
```diff
@@ -175,6 +175,11 @@
   text-overflow: ellipsis;
 }
 
+#CollapsePanel-button {
+  color: white;
+  display: none;
+}
+
 /* Scrollbar styling */
```
Comment (Stegi56) [inline chatez/src/pages/Chats/Components/ChatFeedFriendPanel/ChatFeedFriendPanel.css:183]: comment not needed here, self explanatory

File: chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.tsx [hunk 4]
```diff
@@ -16,7 +16,7 @@ import { renderMessagesByDate } from './ChatFeed.utils';
  * ChatFeed Component
```
Comment (Stegi56) [inline chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.tsx:16]: comment not needed here

File: chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.css [hunk 5]
```diff
@@ -296,42 +296,70 @@
...
+  cursor: pointer;
+  font-size: 16px;
+  padding-left: 10px;
+  color: #666; /* Optional: give some color to the close button */
 }
 
-
 .attachment-preview button:hover {
-    color: #000; /* Optional: add hover effect */
+  color: #000; /* Optional: add hover effect */
```
Comment (Stegi56) [inline chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.css:334]: comment not needed here

File: chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.css [hunk 6]
```diff
@@ -296,42 +296,70 @@
...
-    cursor: pointer;
-    font-size: 16px;
-    padding-left: 10px;
-    color: #666; /* Optional: give some color to the close button */
+  background: none;
+  border: none;
+  cursor: pointer;
+  font-size: 16px;
+  padding-left: 10px;
+  color: #666; /* Optional: give some color to the close button */
```
Comment (Stegi56) [inline chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.css:330]: comment not needed here

**Comment by X2Abdul** [comment]
i'll approve once all asked chnages have been resolved

**Pull request merged**